hubspot/plugin.py

```diff
@@ -1944,3 +1921,35 @@ def hubspot_merge_companies(primary_company_id: str, company_to_merge_id: str):
     """Merge company_to_merge with primary_company, retaining primary company"""
     url = "https://api.hubapi.com/crm/v3/objects/companies/merge"
     _merge_objects(url, primary_company_id, company_to_merge_id)
+
+
+@purpose("Fetch HubSpot List.")
+def hubspot_fetch_list(
```

Comment:
is 'fetch' a term used in the hubspot API?

i'd recommend

hubspot_list_memberships(...)

for this action name.